balboa-agent/src/it/scala/com/socrata/balboa/agent/CustomMatchers.scala

```diff
@@ -0,0 +1,71 @@
+package com.socrata.balboa.agent
+
+import java.net.URL
+import java.util.UUID
+
+import org.joda.time.Duration
+import org.json4s.JsonAST.JLong
+import org.json4s.jackson.JsonMethods.{pretty, render}
+import org.json4s.{JBool, JDouble, JInt, JNothing, JString, JValue}
+import org.scalatest.matchers._
+
+trait CustomMatchers {
```

Comment:
I agree. Do you know of any existing repo serves this repository? Do you mind if I do this another time?